LICENSE.md

```diff
@@ -1,16 +1,16 @@
-Copyright (c) 2005-2015, Zend Technologies USA, Inc.
+Copyright (c) 2005-2017, Zend Technologies USA, Inc.
```

Comment:
We should use a script for these changes. (https://github.com/zendframework/maintainers/issues/6)

Reply:
Nice, haven't seen it. We need then the script there 😄